gramps/plugins/test/test_imports.py

```diff
@@ -243,7 +243,8 @@ def tst(self):
     _tstfiles = []
     for _tstfile in os.listdir(TEST_DIR):
         (fname, ext) = os.path.splitext(os.path.basename(_tstfile))
-        if ext == ".gramps" or ext == ".difs" or ext == ".bak":
+        if ext == ".gramps" or ext == ".difs" or ext == ".bak" \
```

Comment:
I suggest you change:
`ext == ".gramps" or ext == ".difs" or ext == ".bak"`
to:
`ext in (".gramps", ".difs", ".bak")`